cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -98,10 +95,14 @@ if ("${CMAKE_RANLIB}" STREQUAL "")
 	set(CMAKE_RANLIB "${EMSCRIPTEN_ROOT_PATH}/emranlib${EMCC_SUFFIX}" CACHE FILEPATH "Emscripten ranlib")
 endif()
 
-# Don't do compiler autodetection, since we are cross-compiling.
-include(CMakeForceCompiler)
-CMAKE_FORCE_C_COMPILER("${CMAKE_C_COMPILER}" Clang)
-CMAKE_FORCE_CXX_COMPILER("${CMAKE_CXX_COMPILER}" Clang)
+# CMakeForceCompiler is not to be used from CMake 3.5+
```

Comment:
This block and added comment looks good, although you removed the original comment telling why the option is present for older CMakes: "# Don't do compiler autodetection, since we are cross-compiling.". I think that would be good to retain in place.